x-pack/plugins/fleet/server/services/epm/archive/validation.ts

```diff
@@ -278,10 +279,12 @@ function parseAndVerifyVars(manifestVars: any[], location: string): RegistryVars
   }
   return vars;
 }
-function parseAndVerifyPolicyTemplates(manifest: any): RegistryPolicyTemplate[] {
+export function parseAndVerifyPolicyTemplates(
+  manifest: PackageSpecManifest
+): RegistryPolicyTemplate[] {
   const policyTemplates: RegistryPolicyTemplate[] = [];
   const manifestPolicyTemplates = manifest.policy_templates;
-  if (manifestPolicyTemplates && manifestPolicyTemplates > 0) {
+  if (manifestPolicyTemplates && manifestPolicyTemplates.length > 0) {
```

Comment:
@skh i've copied over an am using a lot of the parseAndVerify* functions for now to create the package info from storage.  I've noticed it doesn't match info coming from the registry.  For example policy_templates were not created because the above condition was not being met.  The `default` array in the input vars of a data stream is also not showing up.  Not sure if i'm doing something wrong but if not I assume the uploaded package would have these problems as well as its using that code, too.  Thinking we should do some snapshot testing or something to make sure we're parsing the manifest correctly.